docs/intro/contributing.txt

```diff
@@ -194,12 +197,12 @@ When contributing to Django it's very important that your code changes don't
 introduce bugs into other areas of Django. One way to check that Django still
 works after you make your changes is by running Django's test suite. If all
 the tests still pass, then you can be reasonably sure that your changes
-haven't completely broken Django. If you've never run Django's test suite
-before, it's a good idea to run it once beforehand just to get familiar with
-what its output is supposed to look like.
+work and haven't broken other parts Django. If you've never run Django's test
+suite before, it's a good idea to run it once beforehand to get familiar with
+its output.
 
-Before running the test suite, install its dependencies by first ``cd``-ing
-into the Django ``tests/`` directory and then running:
+Before running the test suite, install its dependencies by ``cd``-ing into the
+Django ``tests/`` directory and then running:
 
 .. console::
 
```

Comment:
the below note about 'if you encounter an error' could be pulled up a bit. Also potentially to ease onboarding of contributors, the requirement on `libmemcached` (and any other default C deps?) could be relaxed by allowing such tests to be skipped if memcached isn't installed and removing it from default requirements.